lib/sinon/util/fake_server.js

```diff
@@ -137,13 +142,21 @@ var fakeServer = {
         }
     },
 
+    logger: function () {
+        // no-op; override via configure()
```

Comment:
The config object is passed to the `fake_server#create` method; this same config object is also passed to the `configureLogError` function in the `fake_server#configure()` method.

If the user passes a `logger` func in their config object it will be set on both the `fake_server` instance (overwriting the default implementation shown here) and also to the `logError` instance created by `fake_server#configure()`.